TimeCapsuleManagement/views.py

```diff
@@ -17,9 +20,32 @@ def my_capsules(request):
     if request.method == 'POST':
         form = CapsuleForm(request.POST, request.FILES)
         if form.is_valid():
-            form.save()
+            capsule = form.save()
+            # Handle the uploaded files for CapsuleContent
+            files = request.FILES.getlist('capsule_contents')
+            for file in files:
+                mime_type, _ = mimetypes.guess_type(file.name)
+                file_type = 'text'  # Default file type
+                if mime_type:
+                    if mime_type.startswith('image/'):
+                        file_type = 'photo'
+                    elif mime_type.startswith('video/'):
+                        file_type = 'video'
+                # Create a CapsuleContent object for each file
+                CapsuleContent.objects.create(
```

Comment:
Try to call operations functions, or else nvm.

Reply:
I will look at how we can do that in a future PR.